scripting/spawnandkillprotection.sp

```diff
@@ -396,20 +396,17 @@ public Event_PlayerDeath(Handle:event, const String:name[], bool:broadcast)
 public Action:Hook_OnTakeDamage(victim, &attacker, &inflictor, &Float:damage, &damagetype, &weapon,
 								Float:damageForce[3], Float:damagePosition[3], damagecustom)
 {	
-	if (isKillProtected[victim]) {
-
+	if (isKillProtected[victim])
+	{
```

Comment:
The { brackets on its own line are used for the functions only. The { } for if else etc. are on the same align.
It's maybe confusing, but I let's keep it consistent with the other code :)
Can you change this? Thanks!

Reply:
No problem. Adapted to your style now. Thank you for the approvals!